pegasus/sites.v3/code.org/views/alt-about_nav.haml

```diff
@@ -1,11 +0,0 @@
-#about_nav
```

Comment:
Woah so this page wasn't being touched at all???

Reply:
I can't find anywhere we render this view and it came it as part of staging content scoop changes 3 years ago. Not only does this view reference the non-existent Diversity Council, it also links to code.org/about/alt-about/our-impact which isn't rendered anywhere (as far as I can tell) either. 🤷‍♀️  I'm fairly confident it's just leftover from some long ago header work, but is there anyone else we should confirm with?

Reply:
We can definitely get rid of this! 😄